stories/topics/con-gcp-pull-remove-container-image.adoc

```diff
@@ -2,7 +2,7 @@
 
 = Pulling the ansible-on-clouds-ops container image
 
-Pull the Docker image for the Ansible on Clouds operational container with the same tag as your foundation deployment.
+Pull the Docker image for the Ansible on Clouds operational container with the same tag as your foundation deployment. If you are unsure of the version you have deployed, see xref:ref-aap-using-playbooks[Command Generator] and playbook *gcp_get_aoc_version* for more information on figuring out the current version of Ansible on Clouds deployment.
```

Comment:
New line before the second sentence, and

for more information on finding the current....